src/node_crypto.cc

```diff
@@ -5631,12 +5640,17 @@ class RandomBytesRequest : public AsyncWrap {
 
   size_t self_size() const override { return sizeof(*this); }
 
+  inline void set_create_buffer(bool create_buffer) {
+    create_buffer_ = create_buffer;
```

Comment:
This isn't exactly wrong but I'd make it an enum (e.g. `enum FreeMode { FREE_DATA, DONT_FREE_DATA }`) that you pass to the constructor and is a const data property from thereon.  You could give it a default value if you don't want to update construct calls everywhere.